### ISSUE TITLE
"no-large-media: * true" doesn't block all videos

### ISSUE THREAD

**Issue by BenFroh** [gorhill/uBlock#2946]
### Describe the issue
I set the custom filter "no-large-media: * true" to prevent Stagefright exploits.
Sadly, on some sites aren't the videos blocked.

(But besides that it would be great to have an option to allow pictures and bock only videos / audios.)

### One or more specific URLs where the issue occurs
http://www.webmfiles.org/demo-files/

### Steps for anyone to reproduce the issue
1. set the custom filter: no-large-media: * true
2. go to the site: http://www.webmfiles.org/demo-files/
3. click on the video at the bottom of the page.

### Your settings
- OS/version: Android 5.0.1
- Browser/version: Firefox 56.0b7
- uBlock Origin version: 1.14.0

##### Your filter lists
Default filter lists + German

##### Your custom filters (if any)
no-large-media: * true

**Comment by gorhill** [comment]
I could not reproduce your issue at first, I had toggled on "no-large-media" before going to the site. The video was properly blocked when I clicked on it, and the logger reported it was blocked because of "no-large-media".

However, I toggled off the switch, and I could play the video -- as expected. I toggled on the switch and I still could play the video -- unexpectedly. Does that resemble to what you did?
